### PR TITLE
Add AwesomePatrol to organization members

### PR DESCRIPTION
**Pull request by AwesomePatrol**
I request to join as an organization member to Cilium project in accordance with https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#organization-member

My contributions:
 * Hubble field mask:
   * https://github.com/cilium/cilium/pull/23198
   * https://github.com/cilium/hubble/pull/1101
 * misc:
   * https://github.com/cilium/hubble-otel/pull/97
   * https://github.com/cilium/hubble-otel/pull/96
   * https://github.com/cilium/cilium/pull/21860
 * in-progress:
   * https://github.com/cilium/cilium/issues/25508
     * https://github.com/cilium/cilium/pull/26379
   * https://github.com/cilium/cilium/pull/26367

I plan to continue contributing and be further involved in the project, especially Hubble.

### CODE REVIEW

**Comment by xmulligan** [comment]
Thanks for your contributions! Happy to approve once you switch this from a draft PR :)

**Comment by AwesomePatrol** [comment]
I wanted to get at least one more minor change merged, before applying. Now it will be handy as I need to rerun some tests 😃

**Review by xmulligan — APPROVED** [review]

**Review by kaworu — APPROVED** [review]

**Review by rolinh — APPROVED** [review]

**Comment by kaworu** [comment]
> Receive two "LGTM" comments from any Org Members, and approval by a committer who merges the PR

This PR has three LGTM/approval from Commiters, merging.

**Pull request merged**
